grafana-dashboard/ClickHouseKeeper_dashboard.json

```diff
@@ -941,14 +941,14 @@
         "allValue": ".+",
         "current": {},
         "datasource": "${DS_PROMETHEUS}",
-        "definition": "label_values(zk_ruok, namespace)",
+        "definition": "label_values(ClickHouseAsyncMetrics_KeeperPacketsSent,namespace)",
         "hide": 0,
         "includeAll": true,
         "label": null,
         "multi": true,
         "name": "namespace",
         "options": [],
-        "query": "label_values(zk_ruok, namespace)",
+        "query": "label_values(ClickHouseAsyncMetrics_KeeperPacketsSent,namespace)",
```

Comment:
lets replace to `label_values(up{container_name="clickhouse-keeper"},namespace}`